src/routes/index.ts

```diff
@@ -47,5 +48,10 @@ const authRouter = Router();
 baseRouter.use("/auth", authRouter);
 authRouter.post("/login", adminLoginLimiter, AuthController.login);
 
+const apiKeyRouter = Router();
+baseRouter.use("/apiKey", apiKeyRouter);
```

Comment:
```suggestion
baseRouter.use("/api-key", apiKeyRouter);
```
Please don't use uppercase letters in url paths

Reply:
I used uppercase because of [ballotID](https://github.com/LeoTuet/backend-we-want-to-sleep/pull/62/files/7bc5f5a811b787a1a3115210e016d706dc4f23aa#diff-4c1a276a826a5147689418cdbf9bde24c8326cedb30a61144fe7867bd99113efR31)

Reply:
I used uppercase because [ballotID](https://github.com/LeoTuet/backend-we-want-to-sleep/blob/main/src/routes/index.ts#L20) is also uppercase.

Reply:
[ballotID](https://github.com/LeoTuet/backend-we-want-to-sleep/blob/main/src/routes/index.ts#L20) is a variable and not in the path

Reply:
oh f...